### PR TITLE
set code coverage

### PR DESCRIPTION
**Pull request by anatoliikryvishyn**

### CODE REVIEW

**Comment by anatoliikryvishyn** [comment]
@serhiir, please review and test [here](https://tada-interlink.herokuapp.com/) and you can see code coverage statistic [here](http://tada-code-coverage.herokuapp.com/).

**Comment by serhiir** [comment]
@anatoliikryvishyn Please test check/uncheck properly, it doesn't work as expected.

File: app/controllers/items_controller.rb [hunk 1]
```diff
@@ -8,11 +8,11 @@ def create
...
   def update
     @item = Item.find(params[:id])
-    if @item.update_attributes(done: item_update_params)
-      render json: { listId: @item.list_id, itemId: @item.id, done: @item.done? }
-    else
-      render status: 400, json: {error: (t 'items.check_error_notification'), id: @item.id, checked: @item.done? }
-    end
+    render status: 200, json: {listId: @item.list_id,
+                               itemId: @item.id,
+                               done: !@item.done?} if @item.update_attributes(done: item_update_params)
```
Comment (serhiir) [inline app/controllers/items_controller.rb]: @anatoliikryvishyn What about the case when `update_attributes` return `false` and nothing is rendered?

Comment (serhiir) [inline app/controllers/items_controller.rb]: @anatoliikryvishyn Also I'm wondering whether any the test fails?

File: test/controllers/items_controller_test.rb [hunk 2]
```diff
@@ -37,10 +40,21 @@ class ItemsControllerTest < ActionController::TestCase
...
     assert_response 200
   end
+
+  test "shouldn't toggle item's completion if wrong item" do
+
+    item = items(:university)
+
+    patch :update, completed: !item.done?, id: 0
+
+    assert_equal item.done?, Item.find(item.id).done?
```
Comment (serhiir) [inline test/controllers/items_controller_test.rb]: @anatoliikryvishyn What is the reason to use real item here?

File: test/controllers/items_controller_test.rb [hunk 3]
```diff
@@ -37,10 +40,21 @@ class ItemsControllerTest < ActionController::TestCase
...
 
     refute_equal item.done?, Item.find(item.id).done?
 
     assert_response 200
   end
+
+  test "shouldn't toggle item's completion if wrong item" do
+
+    item = items(:university)
+
```
Comment (serhiir) [inline test/controllers/items_controller_test.rb]: @anatoliikryvishyn Adjust please to use the factory.

**Comment by anatoliikryvishyn** [comment]
>@anatoliikryvishyn Please test check/uncheck properly, it doesn't work as expected.

@serhiir, fixed, try again.

**Comment by serhiir** [comment]
@anatoliikryvishyn Please do not push until the review is done.

**Comment by serhiir** [comment]
@anatoliikryvishyn Verified.

**Comment by serhiir** [comment]
@anatoliikryvishyn Please take a look at the following comment: https://github.com/anatoliikryvishyn/TaDa/pull/11#discussion_r48412510

**Comment by serhiir** [comment]
@anatoliikryvishyn Done.

**Comment by anatoliikryvishyn** [comment]
@serhiir, updated.

**Pull request merged**